src/Microsoft.NET.Test.Sdk_props

```diff
@@ -1,18 +0,0 @@
-<!--
-***********************************************************************************************
-Microsoft.NET.Test.Sdk.props
-
-WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
-          created a backup copy.  Incorrect changes to this file will make it
-          impossible to load or build your projects from the command-line or the IDE.
-
-Copyright (c) .NET Foundation. All rights reserved. 
-***********************************************************************************************
--->
-
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <!-- Output type for dotnet core test project should be exe. For full CLR it is Library by default. -->
```

Comment:
Comment out with appropriate bug url because of which we doing this temporary fix.